clang/include/clang/Basic/Cuda.h

```diff
@@ -106,6 +106,8 @@ enum class OffloadArch {
   GFX90a,
   GFX90c,
   GFX9_4_GENERIC,
+  GFX940,
+  GFX941,  
```

Comment:
TODO: Add code to enable SYCL offloading to these AMD GPU targets.

Reply:
These targets have been removed in community, we should follow and not add them in our upstream efforts.